Homework2ed/Homework2ed/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Homework2ed
+{
+    class Program
+    {
+
+        static double TetrahedronVolume(string x)
+        {
+            double i = Double.Parse(x);
+            return (i * i) / (6 * Math.Sqrt(2));
+        }
+
+        static double SurfaceAreaTetrahedron(string x)
+        {
+            double i = Double.Parse(x);
+            return i * i * Math.Sqrt(3);
+        }
+        static void Main(string[] args)
+        {
+            string x = Console.ReadLine();
```

Comment:
А вот тут уместна конвертация в число.
```suggestion
            string input = Console.ReadLine();
            var x = double.Parse(input);
```